docs/samples/rollouts/pinned.yaml

```diff
@@ -6,11 +6,11 @@ spec:
   default: 
     predictor:
       tensorflow:
-        storageUri: "gs://mybucket/mymodel-2"
+        storageUri: "gs://kfserving-samples/models/tensorflow/flowers"
   # Defaults to zero, so can also be omitted or explicitly set to zero.
   canaryTrafficPercent: 0
   canary:
     predictor:
       # Canary is created but no traffic is directly forwarded.
       tensorflow:
-        storageUri: "gs://mybucket/mymodel-3"
+        storageUri: "gs://kfserving-samples/models/tensorflow/flowers-2"
```

Comment:
wouldn't it be better to keep the canary section so users can see what it means to pin all traffic to the default model? Removing it would make this sample not much different from the tensorflow one.

What do you mean by start a new iteration? Would I need to make a `flowers-3` model, as I do not think that exists?

Reply:
sorry you are totally right. I read it wrong, this is the example for pinned model I thought was for promoting the canary model to be the default one.

Reply:
added steps and sample on promoting canary model to default